search-ui/src/router/routes.ts

```diff
@@ -47,6 +48,16 @@ export const routes: RouteRecordRaw[] = [
     component: DashboardView,
     meta: {
       requiresAuth: true,
+      breadcrumb: [
```

Comment:
you've already got the breadcrumbs defined in the resources folder so you should use them as is I think and alter this to 
`breadcrumb:[SearchHomeBreadCrumb, SearchDashboardBreadcrumb]`

Reply:
Removed. Thanks!